.gitignore

```diff
@@ -2,4 +2,5 @@
 node_modules
 .DS_Store
 .idea
-.vscode
\ No newline at end of file
+.vscode
+.env
```

Comment:
updating this file is not allowed, you already have .gitgnore file on both front and backend folders